dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScope.kt

```diff
@@ -156,7 +156,9 @@ internal class RumActionScope(
     ) {
         if (sent) return
 
-        if (resourceCount > 0 || errorCount > 0 || viewTreeChangeCount > 0) {
+        val actualType = type
```

Comment:
what is the difference between `actualType` and `type`? IOW why do we need `actualType`?

Reply:
Because the type is a variable property of this instance, it could be changed from another thread. I'm keeping it in a `val` to avoid having inconsistent value in the rest of the method.